### DISCUSSION TITLE
'vscode-insiders' post_install script... Notes: Please move the following directories

### DISCUSSION THREAD

**Discussion by opund** [ScoopInstaller/Scoop#5454] [Help and Questions] [answered]
From "$env:USERPROFILE\.vscode-insiders\extensions" to
"$env:SCOOP\persist\vscode-insiders\data\extensions"
From "$env:APPDATA\Code - Insiders" to "$env:SCOOP\persist\vscode-insiders\data\user-data"
Add Visual Studio Code as a context menu option by running:
'C:\Users\snkript\scoop\apps\vscode-insiders\current\install-context.reg'
For file associations, run
'C:\Users\snkript\scoop\apps\vscode-insiders\current\install-associations.reg'

As a new scooper trying to git it, I got thru %USERPROFILE%\scoop\shims environment variables but can't seem to readily source instructions to be comfortable attempting to execute the above kind of move :/

**Comment by rashil2000** [comment, marked as answer]
Please run these commands, now migration will happen automatically: 

```pwsh
scoop uninstall vscode-insiders -p
scoop update
scoop install vscode-insiders
```

**Reply by opund** [reply]
Thanks, that was easy :) although C:\Users\NAME> reg import C:\Users\NAME\scoop\apps\vscode-insiders\current\install-context.reg
>>ERROR: Error opening the file. There may be a disk or file system error. 

As admin,  same thing. 
No issues on chkdsk via File Explorer > System (C:) Properties > Error Checking 

Not that it's crucial, but maybe a convenient gateway to use VS Code and beyond...
